react/features/base/react/components/web/NavigateSectionListItemContainer.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Container } from './index.js';
+
+/**
+ * Implements a React/Web {@link Component} for displaying an item in a
+ * NavigateSectionList
+ *
+ * @extends Component
+ */
+export default class NavigateSectionListItemContainer extends Component {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+
+    static propTypes = {
+        /**
+         * The children to be displayed within this Link.
+         */
+        children: PropTypes.node
+    };
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <Container>
```

Comment:
Is this component really necessary? It just wraps children in a Container. This can be inlined, can't it?

Reply:
The component has been changed to `NavigateSectionListItem` because there have the organization of RecentList has been changed and now the `NavigateSelectionListItem` is a part of the `renderItem()` function being sent to `SectionList`